src/js/components/StatusBar.js

```diff
@@ -92,9 +92,10 @@ class StatusBar extends React.Component {
         return null;
       }
 
-      let style = {
-        width: `${scale}%`
-      };
+      if (!style) {
+        style = {};
```

Comment:
You can do `let {className, style = {}} = status;` to default assign in the declaration (unless style is actually being assigned `null`, but from what I can tell it is not).